### ISSUE TITLE
E-mail content for BigQuery access need update

### ISSUE THREAD

**Issue by SichengH** [MIT-LCP/physionet-build#1985]
I recently helped a new user access Physionet datasets on BigQuery. And I realized BigQuery no longer has the "pin" button available. However, in the email sent out by PhysioNet Automated System. It says: 

```
You have requested access to use MIMIC-IV v2.2 in GCP BigQuery.

For information on how to access this resource please follow these steps
1. Navigate to: https://console.cloud.google.com/bigquery
2. In the left sidebar, there will be a button saying "+ADD DATA". Click it and pin the project "physionet-data".
3. You should now see the PhysioNet datasets in the left sidebar.
   - If you don't have access to a particular dataset, no information will be displayed when you click on the dataset name (and you may see a spinning circle icon).
   - If you cannot add the project "physionet-data", please access it through this link: https://console.cloud.google.com/bigquery?project=physionet-data&page=dataset
```
There isn't a "PIN" button anymore and I am sure what the link in 3. does (it leads to an empty page).

**Comment by tompollard** [comment]
Thanks Sicheng, yes this needs an update! 

Are you able to suggest some text (or even better, open a pull request)? 

We've had several discussions with users recently about the new process, but right now I don't know exactly what we should be suggesting. e.g. see: https://github.com/MIT-LCP/mimic-code/discussions/1453 and https://github.com/MIT-LCP/mimic-code/discussions/1154

**Comment by alistairewj** [comment]
Closed by @tompollard in #2014
